src/api/app/models/user_block.rb

```diff
@@ -0,0 +1,27 @@
+class UserBlock < ApplicationRecord
```

Comment:
@hellcp-work I'm a bit unsure about the naming... We start having more of these places where something is blocked. Maybe `UserBlockedUser` would be a bit more precise?

Reply:
Fair enough